configs/202310gavi-4/campaign-default/one/TRUE/AGO_campaign-default_100_one.yml

```diff
@@ -0,0 +1,16 @@
+runname: '202310gavi-4'
+country: 'AGO'
+scenario: 'campaign-default'
+clean: TRUE
+incid:
+  num_samples: 100
+  redraw: FALSE
+  use_country_incid_trend: TRUE
+vacc:
+  targeting_strategy: affected_pop
+  num_skip_years: 3
+  ndoses: one
+setting:
+  incidence_rate_trend: TRUE
+  outbreak_multiplier: FALSE
+  random_seed: 3
```

Comment:
I believe random_seed needs to be the same across all scenarios for a single country so that the results will be comparable between them

Reply:
I think there is no specific reason, I saw that in set_all_parameters.R the random seed ranges from 1 to 4 according to the setting (different combinations of the incidence rate trend and outbreak multiplier). I can add a modification where only for the new touchstone we have the same random seed that is a multi-digit number. Does this sound good?